tests/queries/0_stateless/02967_mysql_settings_override.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
+# Tag no-fasttest: requires mysql client
+
+# Tests the override of certain MySQL proprietary settings to ClickHouse native settings
```

Comment:
Tests that certain MySQL-proprietary settings are mapped to ClickHouse-native settings.